synapse/tests/utils.py

```diff
@@ -151,7 +151,7 @@ def indx(self, norm):
 
         ('test:ndef', ('ndef', {}), {}),
         ('test:runt', ('str', {'lower': True, 'strip': True}), {'doc': 'A Test runt node'}),
-
+        ('test:latlong', ('geo:latlong', {}), {}),
```

Comment:
test_ndef and test_nodeprof fail here because they try to load the test model and don't load the geo model